### DISCUSSION TITLE
Clustering seperate datasets based on anchorset

### DISCUSSION THREAD

**Discussion by SarahMyoblast** [satijalab/seurat#4074] [Q&A] [answered]
Hello @satijalab,

I would like to cluster my 2 datasets separately but based on anchors found when doing integration. Is this possible ?
The goal is to compare the two datasets and the differences in the clusters that anchor together.

The integrated UMAP shows that some of the clusters align in both datasets while others present a shift or are absent. 
In order to compare my two datasets, I would like to find all markers of the clusters (not of the pooled/integrated clusters) separated by conditions while still taking into consideration the anchors / anchorset.  

(I tried analyzing each dataset separately to `FindAllMarkers`, but here I want to compare the anchored datasets.)

Thank you

**Comment by timoast** [comment]
Hi, I'm not sure what you mean by "while still taking into consideration the anchors / anchorset."?

Maybe you are trying to find conserved markers across the two datasets? If so I'd suggest trying the `FindConservedMarkers()` function.

**Comment by SarahMyoblast** [comment]
No, I don't mean the conserved markers. I am interested in the opposite. 

I have 2 samples (A and B) that represent different time points during embryonic development.
When I do the integration of the two samples followed by clustering, I can find sub-populations that are present in both.

But, when I do the clustering separately for each sample, I am not able to tell every time which population in sample A corresponds to which one in sample B. That is because the samples don't align (and the markers are evolving because cells are differentiating).

I want to find all markers for all populations in sample A and all markers for all populations in sample B in a clustering that aligns.
That way i can compare cluster 1 in sample A to cluster 1 in sample B etc.. 

Does that makes any sense ?

Thank you

**Comment by timoast** [comment, marked as answer]
I think the analysis you want to do is similar to that shown in the introductory integration vignette: https://satijalab.org/seurat/articles/integration_introduction.html#introduction-to-scrna-seq-integration-1

See the section "Identify differential expressed genes across conditions"

Once you have a consistent set of cell labels across the datasets (integrated clustering, for example), you can use `FindMarkers()` to find differentially expressed genes within a cluster across the two experiments.
